From ec823d4148fe9182c92c6e748228a527e45f0dcf Mon Sep 17 00:00:00 2001
From: Wouter Wijsman <wouter@gridscale.io>
Date: Fri, 7 Oct 2022 13:56:03 +0200
Subject: [PATCH] Use webkit 4.1 when possible

---
 minigalaxy/ui/webkit.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/minigalaxy/ui/webkit.py b/minigalaxy/ui/webkit.py
index c71a7ff3..2572ff75 100644
--- a/minigalaxy/ui/webkit.py
+++ b/minigalaxy/ui/webkit.py
@@ -1,4 +1,7 @@
 import gi
 
-gi.require_version('WebKit2', '4.0')
+try:
+    gi.require_version('WebKit2', '4.1')
+except ValueError:
+    gi.require_version('WebKit2', '4.0')
 from gi.repository import WebKit2  # noqa: E402,F401
